tests/cli/DataProviderFailuresAndExceptionsCest.php

```diff
@@ -43,7 +43,6 @@ public function runTestWithDataProvidersFailureStdout(\CliGuy $I)
     public function runTestWithDataProvidersFailureStderr(\CliGuy $I)
     {
         $I->executeCommand('run -n unit DataProvidersFailureCest 2>&1',false);
-        $I->seeInShellOutput('[Codeception\Exception\TestParseException]');
```

Comment:
Why did you remove some assertions from this test?

Reply:
The format of output in Symfony Console has changed